testsuite/features/step_definitions/command_steps.rb

```diff
@@ -179,6 +179,18 @@
   $command_output, _code = get_target('server').run(command)
 end
 
+When(/^I use spacewalk-common-channel to add all "([^"]*)" channels$/) do |os_product_version|
+  channels_to_synchronize = CHANNEL_TO_SYNC_BY_OS_PRODUCT_VERSION.dig(product, os_product_version)
+  raise ScriptError, "Synchronization error, version type #{os_product_version} in #{product} product not found" unless channels_to_synchronize
```

Comment:
According to this thread,

https://softwareengineering.stackexchange.com/questions/302530/ruby-if-variable-vs-if-variable-nil
```
  if channels_to_synchronize.nil?
```
is a better coding style than 
```
  unless channels_to_synchronize
```

I'm not sure how well accepted this is. It kind of makes sense to me because it uses `.nil?` explicitely.

It might also be a matter of tastes...

Reply:
Yes, it is correct. I am not expecting `false` values so having it that way is better.
Done.